### ISSUE TITLE
minecraft just started not crash, it started, I didn't change anything, I'm launching now and this mod crush everything

### ISSUE THREAD

**Issue by CyberFIowey** [jediminer543/JMT-MCMT#109]
[crash-2022-03-23_06.47.59-fml.txt](https://github.com/jediminer543/JMT-MCMT/files/8329559/crash-2022-03-23_06.47.59-fml.txt)

**Comment by jediminer543** [comment]
Odd I thought I had fixed this bug

This is caused by the filter autogeneration; I'll try to fix this soon

The temporary fix is to delete all autogenerated filters in the MCMT filters directory (config/mcmt-serdes)

**Comment by n0itLT** [comment]
Yeah this is my issue too, messing with MCA Reborn (minecraft comes alive reborn)
